test/spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 RSpec.configure do |config|
   config.verbose_retry = true
   config.display_try_failure_messages = true
-  config.default_retry_count = 3
+  config.default_retry_count = ENV['TEST_RETRY_COUNT'] || 3
```

Comment:
I'd make that default to 0 and set the ENV variable for the travis jobs

Reply:
I agree with Alex :)

Reply:
Thanks, done here: https://github.com/meedan/check-web/commit/c0b1912968d5a3252368ba094eafa89e589b29cf